src/main/app/payment/PaymentClient.ts

```diff
@@ -0,0 +1,160 @@
+import { Logger } from '@hmcts/nodejs-logging';
+import Axios, { AxiosInstance } from 'axios';
+import config from 'config';
+
+import { getServiceAuthToken } from '../auth/service/get-service-auth-token';
+import { CASE_TYPE, DivorceOrDissolution, JURISDICTION } from '../case/definition';
+import type { AppSession } from '../controller/AppRequest';
+
+const logger = Logger.getLogger('payment');
+
+export class PaymentClient {
+  client: AxiosInstance;
+
+  constructor(private readonly session: AppSession, readonly returnUrl: string) {
+    this.client = Axios.create({
+      baseURL: config.get('services.payments.url'),
+      headers: {
+        Authorization: 'Bearer ' + session.user.accessToken,
+        ServiceAuthorization: getServiceAuthToken(),
+        'return-url': returnUrl,
+        'service-callback-url': '',
```

Comment:
is this needed if we its blank?

Reply:
:+1: Looks like it's optional, so I've removed it